eessi/testsuite/hooks.py

```diff
@@ -695,3 +698,33 @@ def _check_always_request_gpus(test: rfm.RegressionTest):
     if FEATURES[ALWAYS_REQUEST_GPUS] in test.current_partition.features and not test.num_gpus_per_node:
         test.num_gpus_per_node = test.default_num_gpus_per_node
         log(f'num_gpus_per_node set to {test.num_gpus_per_node} for partition {test.current_partition.name}')
+
+
+def write_memory_usage(test: rfm.RegressionTest):
+    """
+    Write the memory usage into the job output file if we are in a Slurm job and if cgroups is enabled in Slurm
```

Comment:
Maybe make it even more explicit how this should be called:

```
    @run_after('setup')
    def measure_memory_usage(self):
        """
        Measure memory useage
        """
        hooks.write_memory_usage(self)
```

Reply:
good suggestion, added in https://github.com/EESSI/test-suite/pull/174/commits/317a2a8966e36c053613e04b628d616c8ec6b803
i also renamed the hooks to make it a bit more consistent